src/store.js

```diff
@@ -29,18 +40,213 @@ class AsyncStorageStore {
 
   setItem(key, value) {
     this.hathorMemoryStorage[key] = value;
-    if (key !== 'wallet:data') {
-      // No need to save wallet data (transactions history) on the async storage
-      AsyncStorage.setItem(key, JSON.stringify(value));
+    AsyncStorage.setItem(key, JSON.stringify(value));
+  }
+
+  /**
+   * Clear all items from AsyncStorage.
+   * Optionally we can erase only the wallet keys.
+   * This will keep biometry keys and notification keys for example.
+   *
+   * @param {boolean} [onlyWalletKeys=false] If true, will delete only keys starting with 'wallet'
+   * @returns {Promise<void>}
+   */
+  async clearItems(onlyWalletKeys = false) {
+    const keys = await AsyncStorage.getAllKeys() || [];
+    this.hathorMemoryStorage = {};
+    if (onlyWalletKeys) {
+      await AsyncStorage.multiRemove(keys.filter((key) => key.startsWith('wallet')));
```

Comment:
Maybe remove the keys that start with `wallet:`? I don't remember all the keys we have and if we might add keys that start with "wallet" but not as a prefix.

Reply:
Done